nntrainer/tensor/tensor.h

```diff
@@ -1976,7 +1976,13 @@ class Tensor {
    * @brief     Set fp16 scale factors of the tensor
    * @param[in] scales fp16 scale factors
    */
-  void setScaleFactors16(std::vector<_FP16> scales);
+  void setScaleFactorsFP16(std::vector<_FP16> scales) {
```

Comment:
We need to organize the functions in the same position. 
setScaleFactors () is in tensor.cpp, and this setScaleFactorsFP16() is in tensor.h.
please place them properly for better maintenance.

Reply:
applied!